tests/meltano/cli/test_add.py

```diff
@@ -23,18 +23,19 @@ def patch_hub(self, meltano_hub_service: MeltanoHubService):
             yield
 
     @pytest.mark.parametrize(
-        "plugin_type,plugin_name,default_variant,related_plugin_refs",
+        "plugin_type,plugin_name,default_variant,required_plugin_refs",
         [
             (PluginType.EXTRACTORS, "tap-carbon-intensity", "meltano", []),
             (PluginType.LOADERS, "target-sqlite", "meltanolabs", []),
-            (
+            pytest.param(
                 PluginType.TRANSFORMS,
                 "tap-carbon-intensity",
                 "meltano",
                 [
                     PluginRef(PluginType.TRANSFORMERS, "dbt"),
                     PluginRef(PluginType.FILES, "dbt"),
                 ],
+                marks=[pytest.mark.xfail],
```

Comment:
Unsure what to do with this. Maybe wait for https://github.com/meltano/meltano/pull/5957 to merge.

Reply:
XFail or delete - I think either is fine.

Suggestion here links to the related info as explanation: https://github.com/meltano/meltano/pull/5963/files#r888434730

Either way, that issue will have a marge conflict to resolve - but should be manageable.

Reply:
Alright, happy merge this and help resolve conflicts in #5957